ini/stm32f4.ini

```diff
@@ -737,6 +737,33 @@ extends         = env:STM32F401RE_creality
 debug_tool      = stlink
 upload_protocol = stlink
 
+#
+# STM32F401RE_freeruns  Creality E3 Free-runs Silent Motherboard
+#
+[env:STM32F401RE_freeruns]
+extends                     = stm32_variant
+board                       = genericSTM32F401RE
+board_build.variant         = marlin_STM32F401RE_freeruns
```

Comment:
is it just me that gets errors building until I change the board_build.variant in stm32f4.ini to uppercase?
```
board_build.variant         = MARLIN_STM32F401RE_FREERUNS
```

Reply:
This board has some weird settings...
They added a board file buildroot/share/PlatformIO/boards/marlin_STM32F401RE_freeruns.json but the dont use it when they set board                       = genericSTM32F401RE
Instead they try an force a variant onto genericSTM32F401RE ??

They also reference in ini/stm32f4.ini pre:buildroot/share/PlatformIO/scripts/random-bin.py
Which does not exist


In my view the env:STM32F401RE_freeruns entry should be, this directly uses the MARLIN_STM32F401RE_FREERUNS variant 


```
#
# STM32F401RE_freeruns  Creality E3 Free-runs Silent Motherboard
#
[env:STM32F401RE_freeruns]
extends                     = stm32_variant
board                       = marlin_STM32F401RE_freeruns
board_build.offset          = 0x10000
board_upload.offset_address = 0x08010000
board_build.rename          = firmware-{date}-{time}.bin
build_flags                 = ${stm32_variant.build_flags} -DSTM32F401xE #-DMCU_STM32F401RE #-DSTM32F4
                              -DSS_TIMER=4 -DTIMER_SERVO=TIM5
                              -DTRANSFER_CLOCK_DIV=8
build_unflags               = ${stm32_variant.build_unflags} -DUSBCON -DUSBD_USE_CDC
monitor_speed               = 115200

```